src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs

```diff
@@ -339,6 +339,13 @@ void reportDiagnosticAtLocations(DiagnosticDescriptor descriptor, ImmutableDicti
 
                 void reportDeclareNewApi(ISymbol symbol, bool isImplicitlyDeclaredConstructor, string publicApiName)
                 {
+                    // TODO: workaround for https://github.com/dotnet/wpf/issues/2690
```

Comment:
Do we need to file a tracking issue in roslyn-analyzers repo to revert this change once https://github.com/dotnet/wpf/issues/2690 is fixed? If not, can you add a comment to https://github.com/dotnet/wpf/issues/2690 that this workaround needs to be reverted once that issue is fixed?

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/3345